src/server/ua_subscription_events.h

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2018 (c) Ari Breitkreuz, fortiss GmbH
+ */
+
+#ifndef UA_SUBSCRIPTION_EVENTS_H_
+#define UA_SUBSCRIPTION_EVENTS_H_
+
+#ifdef UA_ENABLE_EVENTS
+
+#define UA_NS0ID_SIMPLEOVERFLOWEVENTTYPE 4035
+
+#ifdef UA_DEBUG_EVENTS
```

Comment:
Can we move this to a unit test?
Test code does not live in the main library atm.

Reply:
In that case there would be no way to have a visual of events working until AuditEvents and the such are implemented since the events can't be seen in UAExpert and none would be generated for the client tutorial on events.